test/performance/mako/sidecar.go

```diff
@@ -31,21 +39,53 @@ const (
 	sidecarAddress = "localhost:9813"
 )
 
+var (
```

Comment:
General go philosophy is that flags are only in main files and the values passed inside.